source/Nevermore.IntegrationTests/Chaos/ChaosSqlCommand.cs

```diff
@@ -20,7 +21,12 @@ public ChaosSqlCommand(DbCommand wrappedCommand, double chaosFactor)
         void MakeSomeChaos()
```

Comment:
`MakeSomeChaos` is working, this is what was breaking this branch: `core/poc-retry-transaction2`